src/components/LoanCards/ExpandableLoanCard/ExpandableLoanCard.vue

```diff
@@ -0,0 +1,127 @@
+<template>
```

Comment:
This file is the core UI that's shared amongst the cards in the row as well as the moving, expanding card